### PR TITLE
Adapt attributes

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | See https://github.com/yiisoft/request-model/pull/48

### CODE REVIEW

File: src/Blog/BlogController.php [hunk 1]
```diff
@@ -72,9 +74,9 @@ public function __construct(
      *    ),
      * )
      */
-    public function index(PageRequest $request, PaginatorFormatter $paginatorFormatter): Response
+    public function index(PaginatorFormatter $paginatorFormatter, #[QueryParam('page')] int $page): Response
```
Comment (samdark) [inline src/Blog/BlogController.php:77]: Looks a bit magical but quite cool.

File: src/Blog/BlogController.php [hunk 2]
```diff
@@ -208,10 +210,10 @@ public function create(EditPostRequest $postRequest, UserRequest $userRequest):
      *     )
      * )
      */
-    public function update(EditPostRequest $postRequest): Response
+    public function update(EditPostRequest $postRequest, #[RouteParam('id')] int $id): Response
```
Comment (samdark) [inline src/Blog/BlogController.php:213]: EditPostRequest not used?

File: src/Blog/BlogController.php (hunk 1 above)
Comment (samdark) [reply src/Blog/BlogController.php:77]: PageRequest isn't used anymore?

File: src/Blog/BlogController.php [hunk 3]
```diff
@@ -135,12 +137,12 @@ public function index(PageRequest $request, PaginatorFormatter $paginatorFormatt
      *    ),
      * )
      */
-    public function view(ViewPostRequest $request): Response
+    public function view(#[RouteParam('id')] int $id): Response
```
Comment (samdark) [inline src/Blog/BlogController.php:140]: ViewPostRequest isn't used anymore?

File: src/Blog/BlogController.php (hunk 2 above)
Comment (rustamwin) [reply src/Blog/BlogController.php:213]: Used.

File: src/Blog/BlogController.php (hunk 1 above)
Comment (rustamwin) [reply src/Blog/BlogController.php:77]: > PageRequest isn't used anymore?

Yes, removed.

File: src/Blog/BlogController.php (hunk 3 above)
Comment (rustamwin) [reply src/Blog/BlogController.php:140]: Removed.

File: src/Blog/PageRequest.php [hunk 4]
```diff
@@ -1,33 +0,0 @@
...
- *          example="2"
- *      ),
- *      in="query",
- *      name="page",
- *      parameter="PageRequest"
- * )
- */
-final class PageRequest extends RequestModel
-{
-    private const DEFAULT_PAGE_PARAM = 1;
```
Comment (samdark) [inline src/Blog/PageRequest.php:23]: Was it lost?

Comment (rustamwin) [reply src/Blog/PageRequest.php:23]: :cry:

**Review by samdark — APPROVED** [review]

**Pull request merged**